airbyte-integrations/connectors/source-instagram/source_instagram/manifest.yaml

```diff
@@ -0,0 +1,732 @@
+version: 1.3.1
```

Comment:
@btkcodedev see? When you make a connector in Builder, it's supposed to use the new CDK ;)

Reply:
Sure! Will prune Docker volumes and fetch new for next one